api/v1/client.go

```diff
@@ -845,6 +849,137 @@ func (c *storageClient) Account(ctx context.Context, r *http.Request) (*Account,
 	return &a, nil
 }
 
+// Delegations returns a list of delegations.
+func (c *storageClient) Delegations(ctx context.Context, r *http.Request) (*DelegationList, error) {
+	chainID, ok := ctx.Value(ChainIDContextKey).(string)
+	if !ok {
+		return nil, common.ErrBadChainID
+	}
+
+	qb := NewQueryBuilder(fmt.Sprintf(`
+			SELECT delegatee, shares, escrow_balance_active, escrow_total_shares_active
+				FROM %s.delegations
+				WHERE delegator = $1::text;
```

Comment:
Ah. I think you might need the `JOIN` after all. `escrow_balance_active` and `escrow_total_shares_active` aren't fields in delegations. My mistake.